week-1-practise/level-2/3-currency-conversion.js

```diff
@@ -5,7 +5,9 @@
   Write a function that converts a price to USD (exchange rate is 1.4 $ to £)
 */
 
-function convertToUSD() {}
+function convertToUSD(priceInPound) {
```

Comment:
Very good! I really like that you have used a function parameter name that makes it really clear how this function should be used. 🏆